exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -64,6 +64,12 @@
   String SPOOLING_BUFFER_MEMORY = "drill.exec.buffer.spooling.size";
   String BATCH_PURGE_THRESHOLD = "drill.exec.sort.purge.threshold";
 
+  // Spill Options common to all spilling operators
```

Comment:
OK, boot-time. Wish Drill had a clear way to distinguish boot-time from session options.